tests/test_signout.py

```diff
@@ -270,3 +339,10 @@ def test_signout_retains_return_path(self, flask_test_client, mock_redis_session
 
         assert response.status_code == 302
         assert response.location == "/service/sso/signed-out/no_token?return_app=post-award-frontend&return_path=%2Ffoo"
+
+        data = {"return_app": "post-award-frontend", "return_path": "/foo"}
+        endpoint = "/sessions/sign-out"
+        response = flask_test_client.post(endpoint, data=data)
+
+        assert response.status_code == 302
+        assert response.location == "/service/sso/signed-out/no_token?return_app=post-award-frontend&return_path=%2Ffoo"
```

Comment:
Can we move the POST testing into a separate test?

Reply:
Done